ansible/roles/test/files/ptftests/dhcpv6_relay_test.py

```diff
@@ -259,7 +259,7 @@ def verify_relayed_solicit_relay_forward(self):
         masked_packet.set_do_not_care_scapy(scapy.layers.dhcp6.DHCP6_RelayForward, "linkaddr")
 
         # Count the number of these packets received on the ports connected to our leaves
-        solicit_count = testutils.count_matched_packets_all_ports(self, masked_packet, self.server_port_indices)
+        solicit_count = testutils.count_matched_packets_all_ports(self, masked_packet, self.server_port_indices, timeout=4.0)
```

Comment:
Probably the `qlen` argument for PTF also need to be increased. Example:
https://github.com/Azure/sonic-mgmt/blob/master/tests/fib/test_fib.py#L282

Reply:
It didn't failed for dhcp relay test cases during PR test running, I think it just happened on my low performance kvm testbed. Not bring in this change.